src/node_file.cc

```diff
@@ -2748,6 +2748,11 @@ static void Mkdtemp(const FunctionCallbackInfo<Value>& args) {
   CHECK_GE(argc, 2);
 
   BufferValue tmpl(isolate, args[0]);
+  const char* ex = "XXXXXX";
```

Comment:
Can this be a `static constexpr`?

Reply:
It seems not...

```
../../src/node_file.cc:2740:31: warning: ISO C++11 does not allow conversion from string literal to 'char *const' [-Wwritable-strings]
  static constexpr char* ex = "XXXXXX";
```

Reply:
Didn't @jasnell mean `static constexpre const char*` ?